cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-july-2023.adoc

```diff
@@ -0,0 +1,157 @@
+[#id-july2023]
+== Features Introduced in July 2023
+
+Learn about the new Compute capabilities on Prisma™ Cloud Enterprise Edition (SaaS) in July 2023.
+
+The host, container, and serverless capabilities on the *Compute* tab are being upgraded starting on July 16, 2023. When upgraded, the version will be 30.03.xxx.
```

Comment:
Build number to be added here > `30.03.xxx`

Reply:
Done